corehq/apps/app_manager/views/releases.py

```diff
@@ -202,7 +202,8 @@ def get_releases_context(request, domain, app_id):
         'prompt_settings_url': reverse(PromptSettingsUpdateView.urlname, args=[domain, app_id]),
         'prompt_settings_form': prompt_settings_form,
         'full_name': request.couch_user.full_name,
-        'can_manage_releases': can_manage_releases(request.couch_user, request.domain, app_id)
+        'can_manage_releases': can_manage_releases(request.couch_user, request.domain, app_id),
+        'can_edit_apps': request.couch_user.can_edit_apps(),
```

Comment:
@AmitPhulera sorry if this is repeated question from our offline chats. Why do we need `can_edit_apps`?
Should not we just use `can_manage_releases`?

Reply:
Because the View class https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/views/releases.py#L102 is `checking can_edit_apps` permission rather than `can_manage_releases` so we want to have the same checks on hiding the link as well. Otherwise, it will create a scenario where user is displayed the link but when he clicks it he will get a 403.

Reply:
Cool, thanks for taking out time to getting me aligned on this.

Reply:
No worries 😄